weave/wandb_file_manager.py

```diff
@@ -52,7 +52,7 @@ def _local_path_and_download_url(
     if manifest_entry is None:
         return None
     md5_hex = hashutil.b64_to_hex_id(hashutil.B64MD5(manifest_entry["digest"]))
-    base_url = weave_env.wandb_base_url()
+    base_url = weave_env.wandb_frontend_base_url()
```

Comment:
this returns urls to download artifacts so we need the frontend url base to download from frontend. This is the most questionable change

Reply:
This is right for the general case.  But i think your intuition is correct, there's probably a subset of call sites that can use the private url